server/src/views/frame.js

```diff
@@ -283,7 +281,7 @@ class Head extends React.Component {
         {oembed}
         {ogTitle}
         {ogImage}
-        <script nonce={ this.props.cspNonce } dangerouslySetInnerHTML={{__html: `var CONTENT_HOSTING_ORIGIN = "${postMessageOrigin}";`}}></script>
+        <script src="/set-content-hosting-origin.js" />
```

Comment:
No, the nonce is still used in a few places where strings are concatenated with scripts in them (in the react bootstrap process). But it is not used anywhere in jsx any more.